### DISCUSSION TITLE
Future of React Support

### DISCUSSION THREAD

**Discussion by ashu12chi** [ueberdosis/tiptap#1418] [General Questions] [answered]
Hello everyone,

I tried tiptap and I really loved your work. I am planning to build my own rich text editor in react using tiptap framework. Can you please tell me about your future plans regarding React support? How you guys have planned about including new react features? I am asking this because at this stage still in documentation React examples are missing and tiptap is fairly new for React.

Thanks a lot!
  ❤

**Comment by hanspagel** [comment, marked as answer]
I think you’ve answered your question already. 😅 tiptap 1 was Vue-only and we decided to make it framework-agnostic and add official support for React with v2. It’s new, that’s why there is still a lot to do, but the development and support is already on the same level. ✌️

**Reply by ashu12chi** [reply]
Hello @hanspagel 

Thanks for replying. Can you share your plan regarding stable release of v2 and any roadmap document you have planned? One more thing, is there any official slack channel of tiptap?

**Reply by hanspagel** [reply]
There is no roadmap, not even a secret one. We’re working on the things that we think are important and that changes over time, depending on the feedback we receive on all channels.

I don’t think it would help anyone to set a fixed release date for the stable version. We just don’t know what is needed to make it officially stable. That said, huge companies (for example GitLab) are using the tiptap 2 beta in production already. No reason to worry about the beta flag. Just don't be upset if we’ll rename a command on our way to a stable release.

Join our Discord server here: https://discord.gg/WtJ49jGshW
